plugins/gcov/gcov.rake

```diff
@@ -160,21 +160,47 @@ namespace UTILS_SYM do
 
     filter = @ceedling[:configurator].project_config_hash[:gcov_html_report_filter] || GCOV_FILTER_EXPR
 
-    if @ceedling[:configurator].project_config_hash[:gcov_html_report_type] == 'basic'
-      puts "Creating a basic html report of gcov results in #{GCOV_ARTIFACTS_FILE}..."
-      command = @ceedling[:tool_executor].build_command_line(TOOLS_GCOV_POST_REPORT_BASIC, [], filter)
-      @ceedling[:tool_executor].exec(command[:line], command[:options])
-    elsif @ceedling[:configurator].project_config_hash[:gcov_html_report_type] == 'detailed'
-      puts "Creating a detailed html report of gcov results in #{GCOV_ARTIFACTS_FILE}..."
-      command = @ceedling[:tool_executor].build_command_line(TOOLS_GCOV_POST_REPORT_ADVANCED, [], filter)
-      @ceedling[:tool_executor].exec(command[:line], command[:options])
+    if @ceedling[:configurator].project_config_hash[:gcov_html_report].nil?
+      puts "In your project.yml, define: \n\n:gcov:\n  :html_report:\n\n to true or false to refine this feature."
+      puts "For now, assuimg you want an html report generated."
```

Comment:
Typo: `assuimg` should be `assuming`. (Commenting about a typo since it's in a print statement.)

Reply:
fixed